public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -152,6 +153,21 @@ export class GeomapPanel extends Component<Props, State> {
       this.layers = layers;
       this.doOptionsUpdate(0);
     },
+    updateLayer: (uid: string, updatedLayer: MapLayerState<any>) => {
```

Comment:
this is fine for now -- once we can drop uid and use the name as a unique ID, we can pass in the `MapLayerOptions` rather than the state instance